docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -85,8 +85,7 @@ spans than `/api/happy`, for example.
 
 The `count_over_time()` function counts the number of matching spans per time interval.
 The time interval that the count will be computed over is set by the `step` parameter.
-For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
-
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics).
```

Comment:
These changes can be discarded, they make no difference to the final URL.

Reply:
```suggestion
For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
```